Branch-SDK/src/main/java/io/branch/referral/DeviceInfo.java

```diff
@@ -230,6 +230,11 @@ void updateRequestWithV2Params(ServerRequest serverRequest, PrefHelper prefHelpe
                 userDataObj.put(Defines.Jsonkey.OSVersionAndroid.getKey(), SystemObserver.getOSVersion());
             }
 
+            AttributionReportingManager attributionManager = AttributionReportingManager.INSTANCE;
```

Comment:
Can `AttributionReportingManager.INSTANCE` ever be null?

Reply:
I believe references to object's cannot be null